crates/wit-parser/src/ast/resolve.rs

```diff
@@ -129,6 +132,15 @@ impl<'a> Resolver<'a> {
                 }
             }
             self.package_name = Some(cur_name);
+
+            // At most one 'package' item can have doc comments.
+            let docs = self.docs(&cur.docs);
+            if docs.contents.is_some() {
+                if self.package_docs.contents.is_some() {
+                    bail!("found doc comments on multiple 'package' items");
```

Comment:
Could this use `Error` to provide a `Span` to attach this error to?

Reply:
I don't have the comment span here; I guess I could use the package item span?

Reply:
Yeah I think it'd be possible in theory to retain the span of documentation, but otherwise pointing at the `package` item I think is fine

Reply:
I'll give it a shot; it's just awkward because _all_ preceding comments get stored in `ast::Docs` and doc-comments aren't extracted until here in `Resolver::docs`.